examples/article-access.amp.html

```diff
@@ -179,11 +179,11 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section class="login-section">
+        <section amp-access="NOT access" class="login-section">
           <a on="tap:amp-access.login">Login to read more!</a>
         </section>
 
-        <div class="article-body" itemprop="articleBody">
+        <div amp-access="access" class="article-body" itemprop="articleBody">
```

Comment:
Can we change this to something like `allow`/`deny`? It's not clear to me whether `access` blocks should allow access for unregistered users or `NOT access` blocks do it.